### PR TITLE
Add high threshold and low threshold arguments in PSTR

### LINKED ISSUE #222: In PSTR the arguments `low/high_threshold` are missing

### PR DESCRIPTION
**Pull request by kalashsinghal**
closes #222 

@lindadelacombaz It would be great that you review this PR as it's a very small modification in PSTR. 

cc @maurolepore

### CODE REVIEW

**Review by lindadelacombaz — APPROVED** [review]
LGTM ! :) thanks @kalashsinghal.

File: R/pstr.R [hunk 1]
```diff
@@ -5,6 +5,10 @@
 #'
 #' @param companies A dataframe like [pstr_companies].
 #' @param scenarios A dataframe like [pstr_scenarios].
+#' @param low_threshold A numeric value to segment low and medium reduction
+#'   targets.
+#' @param high_threshold A numeric value to segment medium and high reduction
+#'   targets.
```
Comment (maurolepore) [inline R/pstr.R:11]: Well done Kalash!

You may also use this to remove duplication and ensure the definition is always the same.

```r
#' @inheritParams xctr
```

File: R/pstr.R [hunk 2]
```diff
@@ -28,19 +32,19 @@
 #'
 #' # Same
 #' pstr(companies, scenarios)
-pstr <- function(companies, scenarios) {
+pstr <- function(companies, scenarios, low_threshold = 30, high_threshold = 70) {
```
Comment (maurolepore) [inline R/pstr.R:35]: In xctr we use 1/3 and 2/3. Do you plan to do that in a separate PR?

**Review by maurolepore — APPROVED** [review]
LGTM with comments:

Please add a test to ensure the new arguments work. You may copy-paste-modify the tests from xctr.

Also, see and address my comments.

BTW, how can we ensure you remember to change the PR out from Draft mode? Maybe you need a checklist or something?

By forgetting the draft mode is loosing its value. I find it useful as a way to communicate that I'm working on something but I don't want anyone to still do anything with it. That allows me to for example force-push to it -- knowing I'm not affecting anyone.

File: R/pstr.R (hunk 1 above)
Comment (kalashsinghal) [reply R/pstr.R:11]: @maurolepore I apologise for forgetting to keep the PR in the draft mode. The tests take time to finish and then I shift my focus to other work until tests are not complete. That's the reason why I forget to convert the PR from draft mode. I will not repeat this mistake next time.

The definition of `low_threshold` & `high_threshold` is different in case of pstr, hence I would like to keep it that way, but if you would insist to inherit the definitions from xctr then I would be happy to do so.

File: R/pstr.R (hunk 2 above)
Comment (kalashsinghal) [reply R/pstr.R:35]: @maurolepore I can see the `reductions` variable is not between 0 and 1. Hence the thresholds' values cannot be changed to 1/3 and 2/3. Please @lindadelacombaz correct me if I am wrong

Comment (lindadelacombaz) [reply R/pstr.R:35]: Yes true. I have asked @Tilmon if the reduction column are in percentage, and they are, so I have used 30% and 70% values.

Comment (kalashsinghal) [reply R/pstr.R:35]: @maurolepore I will resolve this problem through this [issue](https://github.com/2DegreesInvesting/tiltIndicator/issues/165).
